ansible_collections/arista/avd/roles/eos_designs/docs/tables/platform-settings.md

```diff
@@ -120,10 +120,18 @@
         - feature_support:
             poe: true
             queue_monitor_length_notify: false
+          management_interface: Management0
           platforms:
           - '750'
           - '755'
           - '758'
+          reload_delay:
```

Comment:
Sorry for pointing you in this direction. The default value is defined in `ansible_collections/arista/avd/roles/eos_designs/schema/schema_fragments/platform_settings.schema.yml`. After updating the fragment you need to run `pre-commit run schemas --all` to regenerate schemas and documentation.

Reply:
No worries—thank you for the guidance!